src/build.ts

```diff
@@ -63,32 +67,40 @@ async function build(): Promise<void> {
 
     const ioType = process.env.AZLE_VERBOSE === 'true' ? 'inherit' : 'pipe';
 
-    if (command === 'install-dfx-extension') {
-        handleInstallDfxExtensionCommand(ioType);
+    if (command === 'extension') {
+        const subCommand = process.argv[3];
```

Comment:
We need error handling for sub commands, keep it simple

Reply:
done